tests/integration/targets/test_zabbix_authentication/tasks/zabbix_authentication_test_passwd_check_rules.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Start with a known state with one rule enabled
+  community.zabbix.zabbix_authentication:
+    passwd_check_rules: "avoid_easy_to_guess"
+
+- name: Set the login result based on current zabbix version
+  set_fact:
+    login_result: "{{ check_login_under64_result if zabbix_version is version('6.4', '<') else check_login_result }}"
```

Comment:
Where do you set `check_login_result` first time for >= 6.4?

Reply:
It's defined in the setup_zabbix tests that run before each module int test.

https://github.com/ansible-collections/community.zabbix/pull/1388/files#diff-7e6d484bfeae8442bfb90a404ff23d299f5b2e85d472301e8d1aa26f1f57f4a1